vtex-sites/starter.store
# chore: Updates with `Toast` component

## What's the purpose of this pull request?

Updates the core version adding `Toast` component .

## How to test it?

Use [deploy preview](https://preview-22--starter.preview.vtex.app/) to test. Everything should be the same as before.

### Faststore related PRs

https://github.com/vtex/faststore/pull/1672
